spec/requests/groups_controller_spec.rb

```diff
@@ -1245,6 +1245,28 @@ def expect_type_to_return_right_groups(type, expected_group_ids)
             count: 3
           ))
         end
+
+        it 'display error when try to add to many users at once' do
+          GroupsController.send(:remove_const, "ADD_MEMBERS_LIMIT")
+          GroupsController.const_set("ADD_MEMBERS_LIMIT", 4)
```

Comment:
We need to restore the original constant at the end of the task so that this does not cause a state leak.